src/python/grpcio_health_checking/grpc_health/v1/health.py

```diff
@@ -139,7 +190,7 @@ def Watch(self, request, context, send_response_callback=None):
             )
         return blocking_watcher
 
-    def set(self, service, status):
+    def set(self, service: str, status: int) -> None:
```

Comment:
status was marked as `int`, but it says it's `enum` in doc string, we should keep them the same.

Reply:
updated doc